backend/src/users.rs

```diff
@@ -0,0 +1,147 @@
+use std::fmt;
+
+use crate::FlukeDb;
+use rocket::serde::{json::Json, Deserialize, Serialize};
+use rocket::{fairing::AdHoc, routes};
+use rocket_db_pools::{sqlx, Connection};
+use sqlx::FromRow;
+
+type Result<T, E = rocket::response::Debug<sqlx::Error>> = std::result::Result<T, E>;
+
+// Matches fields from user registration form
+// In the future, hash password before storage
```

Comment:
pw should be hashed on FE so never sent over the network.  You can remove this comment and add a github issue to do this.